### PR TITLE
Initialized benchmarking

### PR DESCRIPTION
**Pull request by 07jasjeet**
Set up benchmarking

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -47,6 +47,12 @@ android {
             applicationIdSuffix ".debug"
             versionNameSuffix ".debug"
         }
+        benchmark {
+            signingConfig signingConfigs.debug
+            matchingFallbacks = ['release']
+            manifestPlaceholders["sentryDsn"] = ""
```
Comment (akshaaatt) [inline app/build.gradle:53]: Why is this needed here?

File: benchmark/build.gradle [hunk 2]
```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'com.android.test'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'org.listenbrainz.android.benchmark'
+    compileSdk 33
+
+    compileOptions {
```
Comment (akshaaatt) [inline benchmark/build.gradle:10]: This part is not needed

**Review by akshaaatt — CHANGES_REQUESTED** [review]
I'm not sure of how this setup is being done. Could you please share the links you referred to for this?

File: app/build.gradle (hunk 1 above)
Comment (07jasjeet) [reply app/build.gradle:53]: All this stuff is IDE generated i just added some bits to make solve some errors

**Comment by akshaaatt** [comment]
Closing this PR for now. Can revisit in future.
